editions/tw5.com/tiddlers/messages/WidgetMessage_ tm-modal.tid

```diff
@@ -14,6 +14,15 @@ The "currentTiddler" variable is set to the title of the modal tiddler, but can
 
 The modal message is usually generated with the ButtonWidget. The modal message is handled by the TiddlyWiki core.
 
+! The Modal Tiddler
+Within the Modal tiddler itself;
```

Comment:
There should be a blank line after the heading.

The word "Modal" should only be capitalised within the heading

The semicolon should be a colon.